WebDriverAgentLib/Utilities/FBXPath.m

```diff
@@ -531,25 +531,23 @@ + (NSString *)name
 
 @implementation FBIndexAttribute
 
-- (instancetype)initWithValue:(NSString *)value
-{
-  self = [super initWithElement:nil];
-  if (self) {
-    _indexValue = value;
-  }
-  return self;
-}
-
 + (NSString *)name
 {
   return kXMLIndexPathKey;
 }
 
-- (NSString *)value
++ (int)recordWithWriter:(xmlTextWriterPtr)writer forValue:(NSString *)value
 {
-  return self.indexValue;
+  if (nil == value) {
```

Comment:
this value can never be equal to null. Also, the returned value here is always expected to be unique for each node